package.json

```
@@ -70,5 +71,9 @@
   },
   "engines": {
     "node": ">=18"
+  },
+  "publishConfig": {
+    "access": "public",
+    "provenance": true
   }
```

It's not necessary to specify `provenance` here since the workflow uses `NPM_CONFIG_PROVENANCE: true`.
With it in the `package.json` it gets really annoying if you need to run a canary release from a local machine and npm won't let you 😅 
```suggestion
  "publishConfig": {
    "access": "public"
  }
```